cinema/urls.py

```diff
@@ -1 +1,21 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (GenreViewSet,
+                          ActorViewSet,
+                          CinemaHallViewSet,
+                          MovieViewSet,
+                          MovieSessionViewSet)
+
+router = routers.DefaultRouter()
+router.register("genres", GenreViewSet)
+router.register("actors", ActorViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+urlpatterns = [
+    path("", include(router.urls))
```

Comment:
it's also convenient to write urlpatterns = router.urls (just for your information :)